protos/pbftpb/pbftpb.proto

```diff
@@ -154,20 +154,28 @@ message Event {
 
   oneof type {
     option (mir.event_type) = true;
-
-    uint64             propose_timeout         = 1;
+    ProposeTimeout             propose_timeout         = 1;
     pbftpb.VCSNTimeout view_change_sn_timeout  = 2;
-    uint64             view_change_seg_timeout = 3;
+    ViewChangeSegTimeout             view_change_seg_timeout = 3;
```

Comment:
Alignment inconsistent with the rest of the protobuf definitions. (Elsewhere as well, not further repeating this comment.)

Reply:
fixed